runtime/context.go

```diff
@@ -0,0 +1,30 @@
+package runtime
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/metadata"
+	"net/http"
+	"strings"
+)
+
+const metadataHeaderPrefix = "X-Grpc-Metadata-"
+
+/*
+Add context information such as metadata from the request.
```

Comment:
http://golang.org/doc/effective_go.html#commentary

The comment should be a complete sentence and should begin with "AnnotateContext adds ...".